validator/testcases/javascript/predefinedentities.py

```diff
@@ -88,7 +131,59 @@
                  "version of Firefox. It should not be referenced in any "
                  "code."}
 
+
+ADDON_INSTALL_METHOD = {
+    "value": {},
+    "dangerous": {
+        "description": (
+            "Add-ons may install other add-ons only by user consent. Any "
+            "such installations must be carefully reviewed to ensure "
+            "their safety."),
+        "editors_only": True,
+        "signing_severity": "high"},
+}
+
+
+SEARCH_MESSAGE = "Potentially dangerous use of the search service"
+SEARCH_DESCRIPTION = (
+    "Changes to the default and currently-selected search engine settings "
+    "may only take place after users have explicitly opted-in, by taking "
+    "a non-default action. Any such changes must be reverted when the add-on "
+    "making them is disabled or uninstalled.")
+
+def search_warning(severity="medium", editors_only=False,
+                   message=SEARCH_MESSAGE,
+                   description=SEARCH_DESCRIPTION):
```

Comment:
You aren't using any other parameters than `severity` in your calls, do we still need those extra ones here? Are you planning on using them in this PR?

I'd advise against premature optimization.